src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/runInPodWithRestartWithLongSleep.groovy

```diff
@@ -0,0 +1,24 @@
+package org.csanchez.jenkins.plugins.kubernetes.pipeline
+
+podTemplate(label: 'mypod', containers: [
+        containerTemplate(name: 'busybox', image: 'startx/fedora', ttyEnabled: true, command: '/bin/cat'),
+]) {
+
+    node ('mypod') {
+        stage('Run') {
+            container('busybox') {
+                    semaphore 'wait'
+                    sh '''
+                    #!/bin/bash
+                    for i in 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
```

Comment:
sh 'for i in `seq 1 20`; do echo $i; sleep 5; done'

Reply:
I was having issues with 'seq' at one time...I thought it was due to the busybox image. I will update my sh call.